frontend/src/components/NewIssueModal/NewIssueModal.tsx

```diff
@@ -72,6 +74,8 @@ const NewIssueModal: React.FC<React.PropsWithChildren<NewIssueModalProps>> = ({
 	}, [defaultIssueTitle, commentText])
 
 	const [containerId, setContainerId] = useState('')
+	const [issueTypeId, setIssueTypeId] = useState('')
```

Comment:
For most other integrations the containerId was used to set the issueTeamId.
In the case of Jira Integration, we needed two fields: issueTypeId and issueProjectId.
This was how I set them.